packages/app/src/app/pages/Sandbox/Editor/Content/Tabs/Tab/Tab.tsx

```diff
@@ -0,0 +1,71 @@
+import React, { useState } from 'react';
+import ContextMenu from 'app/components/ContextMenu';
+import { Container, TabTitle, StyledCloseIcon } from './elements';
+
+export const Tab = ({
+  active,
+  dirty,
+  isOver,
+  position,
+  tabCount,
+  closeTab,
+  onClick,
+  onDoubleClick,
+  children,
+  title,
+  items,
+}) => {
+  const [isHovering, setHovering] = useState(false);
+
+  const handleMouseEnter = () => setHovering(true);
+
+  const handleMouseLeave = () => setHovering(false);
+
+  const handleCloseTab = e => {
+    e.preventDefault();
+    e.stopPropagation();
+
+    if (closeTab) {
+      closeTab(position);
+    }
+  };
+
+  const onMouseDown = e => {
+    if (e.button === 1) {
+      // Middle mouse button
+      handleCloseTab(e);
+    }
+  };
+
+  const renderTabStatus = () => {
+    if (isHovering && tabCount > 1) {
```

Comment:
This can be simplified by just calculating the `show` prop

```jsx
const show = isHovering && tabCount > 1 ? 'true' : undefined;

return <StyledCloseIcon onClick={handleCloseTab} show={show} />;
```